### PR TITLE
New design for home page with filters

### LINKED ISSUE #4: New design for the header
must contain:
- [ ] align the title and the text to the left of the section
- [ ] add an image to the right of the section
- [ ] add background color and round corners
- [ ] add different  icons for each filter button
- [ ] add the background color and round corners for the filter buttons
- [ ] when a filter button is clicked change the background color of the section above and the background color of the button

[figma link](https://www.figma.com/proto/Fm1mdnskOPnJCX1AWgpR3U/Ajut.md_Web-Design_UI?scaling=min-zoom&node-id=1%3A2)

image

### PR DESCRIPTION
**Pull request by vitaliel**
It depends on https://github.com/code4moldova/ajut.md/pull/34

Closes #4

It looks like:
Screenshot 2021-10-26 at 17-42-57 Luptă împotriva pandemiei de Covid-19

### CODE REVIEW

File: ro_help/ro_help/settings/base.py [hunk 1]
```diff
@@ -211,13 +207,12 @@
...
 # SMTP
-EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
-
 EMAIL_CONFIG = env.email_url("EMAIL_URL", default="smtp://user:password@localhost:25")
 vars().update(EMAIL_CONFIG)
 
-DEFAULT_FROM_EMAIL = "noreply@rohelp.ro"
+EMAIL_BACKEND = env.str("EMAIL_BACKEND", "django.core.mail.backends.smtp.EmailBackend")
 
+DEFAULT_FROM_EMAIL = env.str("DEFAULT_FROM_EMAIL", "noreply@rohelp.ro")
```
Comment (alexkingdom) [inline ro_help/ro_help/settings/base.py]: @johnsmithm I think we need to adjust the email in all places, what do you think ?

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1328,8 +1264,8 @@ msgid ""
 "The password reset link was invalid, possibly because it has already been "
 "used. Please request a new password reset."
 msgstr ""
-"Ссылка для сброса пароля недействительна, возможно, потому, что она уже была"
-"использована. Запросите новый сброс пароля."
+"Ссылка для сброса пароля недействительна, возможно, потому, что она уже "
+"былаиспользована. Запросите новый сброс пароля."
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1909,8 +1845,9 @@ msgid ""
 "For any questions or issues, as well as to ensure your rights in regards to "
 "the processing of personal data by Code for Romania, you can contact us at:"
 msgstr ""
-"По любым вопросам или проблемам, а также для обеспечения ваших прав в отношении"
-"обработки персональных данных Code for Moldova, вы можете связаться с нами по адресу:"
+"По любым вопросам или проблемам, а также для обеспечения ваших прав в "
+"отношенииобработки персональных данных Code for Moldova, вы можете связаться "
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -1994,9 +1931,9 @@ msgid ""
 "                "
 msgstr ""
 "\n"
-"                    Платеж за %(ngo)s подтвержден! Спасибо"
-"за вашу поддержку в этот период! Любая помощь увеличивает наши возможности"
-"делать добро дальше. \n"
+"                    Платеж за %(ngo)s подтвержден! Спасибоза вашу поддержку "
+"в этот период! Любая помощь увеличивает наши возможностиделать добро "
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -2009,9 +1946,9 @@ msgid ""
 "                    "
 msgstr ""
 "\n"
-"                        Платеж за %(ngo)s обрабатывается."
-"Спасибо за вашу поддержку в это время! Любая помощь увеличивает нашу"
-"способность творить добро дальше.\n"
+"                        Платеж за %(ngo)s обрабатывается.Спасибо за вашу "
+"поддержку в это время! Любая помощь увеличивает нашуспособность творить "
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -1145,13 +1070,13 @@ msgid ""
...
-"Ajut.md - это полнофункциональная цифровая платформа, которая может быть использована"
-"всеми организациями, которые активно участвуют в предотвращении распространения"
-"Covid-19. Основная цель палтформы - помочь организациям собрать"
-"ресурсы, которые им нужны. Платформа поможет вам найти больше всего"
-"насущные потребности, с которыми сталкиваются организации по всей стране. В результате, это"
-"облегчает вам выбор дела, которому вы жертвуете, или усилий, которые вы делаете"
-"способствовать."
+"Ajut.md - это полнофункциональная цифровая платформа, которая может быть "
+"использованавсеми организациями, которые активно участвуют в предотвращении "
+"распространенияCovid-19. Основная цель палтформы - помочь организациям "
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo ?

**Review by alexkingdom — CHANGES_REQUESTED** [review]

**Comment by alexkingdom** [comment]
Basically seems all is good just some typo errors

**Comment by vitaliel** [comment]
@alexkingdom typos are from PR 34 that I used as base. I fixed them and when 34 will be merged, this PR should look fine.

**Review by alexkingdom — APPROVED** [review]

**Comment by alexkingdom** [comment]
@vitaliel Please update your PR and fix conflicts 😃

**Comment by vitaliel** [comment]
@alexkingdom fixed, it was binary file of gettext. I'm not a fun of storing generated mo files in repository.

**Pull request merged**